src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+import './Footer.scss';
+
+const backToTop = () => {
+  window.scrollTo({
+    top: 0,
+    behavior: 'smooth',
+  });
+};
+
+export const Footer: React.FC = () => (
+  <footer className="footer">
+    <div className="footer__container">
+      <div className="footer__logo">
+        <a href="/" className="footer__logo__link">
+          <img
+            src={require('../../images/icons/Logo.svg').default}
```

Comment:
Ask Semen how he pasted svg images without require.
And also try to avoid such rules 
`/* eslint-disable @typescript-eslint/no-var-requires */`
`/* eslint-disable global-require */`